go.mod

```
@@ -1,6 +1,8 @@
 module github.com/rancher/elemental-toolkit/v2
 
-go 1.22
+go 1.22.0
```

do we need the .0 part? If it is for the CodeCov thing, that is solved now 🎉 
... otherwise just ignore this 😅 
```suggestion
go 1.22
```

---

I did not add manually this change (must a consequence of `go mod venor` or `go mod tidy`). I any case makes sense keeping the version without the patch level, thanks.